libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/DatasetCohortStatsTable.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  defaultModelAssessmentContext,
+  ModelAssessmentContext
+} from "@responsible-ai/core-ui";
+import { IDropdownOption } from "office-ui-fabric-react";
+import React from "react";
+
+import { CohortStatsHeatmap } from "./CohortStatsHeatmap";
+
+interface IDatasetCohortStatsTableProps {
+  selectableMetrics: IDropdownOption[];
+  selectedMetrics: string[];
+}
+
+class IDatasetCohortStatsTableState {}
+
+export class DatasetCohortStatsTable extends React.Component<
```

Comment:
I'm terrified to mention that there are no tests for this new code, but then if I wrote this class I wouldn't have added them either, so you can just ignore this.  But it would be nice to add some e2e tests and perhaps at some point even we can just add simple unit tests in later PRs.

Reply:
Certainly! The only reason I even put out the PR is because it's behind a feature flag. I have an item for writing a whole series of e2e tests, but I wanted to keep the PRs small. The number of comments on this PR tells me that making PRs small is a great idea if you want feedback. Love the energy and will keep making small changes. Don't worry, I've made sure to budget for e2e and I will not remove the feature flag unless we have full coverage.